fastlane/Scanfile

```diff
@@ -1,5 +1,5 @@
 scheme "Auth0.iOS"
-devices ["iPhone 8"]
+devices ["iPhone 8 (11.2)", "iPhone 8 (12.0)", "iPhone 7 (10.3.1)"]
```

Comment:
I see what you're doing for your own tests, but please don't include in this PR. At best it would be in a different PR, although I intend to replace with multiple platform tests, which i recently added to Lock.swift.